source/dub/commandline.d

```diff
@@ -495,7 +497,8 @@ class InitCommand : Command {
 			p.description = input("Description", p.description);
 			p.authors = input("Author name", author).split(",").map!(a => a.strip).array;
 			p.license = input("License", p.license);
-			p.copyright = input("Copyright string", p.copyright);
+			string copyrightString = .format("Copyright © %s, %-(%s, %)", Clock.currTime().year, p.authors);
```

Comment:
Too few people seem to know this useful formatting behavior, see docs and examples at https://dlang.org/phobos/std_format.html#.formattedWrite.